### ISSUE TITLE
Upgrade OpenGL requirement to 4.x ?

### ISSUE THREAD

**Issue by nyue** [PixarAnimationStudios/OpenSubdiv#887]
Hi,

  Given that the OpenGL examples fails to run with OpenGL 3.3 shaders etc and no one on the dev team has OpenGL 3 boxes to verify the build/code, it might be better to just flag OpenSubdiv as requiring OpenGL 4

  Labelling it OpenGL 3.3 compatible does not sound right.

Cheers

**Comment by significant-bit** [comment]
Wait!!! We're in the middle of an upgrade to OpenGL 3.3 for Blender 2.8 (release next year). Enabling OpenSubdiv across Mac/Windows/Linux is a big part of that. I can test and report issues with GL 3.3.

Don't mind if the examples need GL 4.x as long as the core library supports 3.3+.

**Comment by nyue** [comment]
Yes, I think making the examples require OpenGL 4.x should be sufficient as they are the ones running the shader code. The shader code loaded if one builds with OpenGL 3.3 has lots of errors and basically the examples plainly does not work.

**Comment by jtran56** [comment]
Filed as internal issue #151673.
